spec/models/order_spec.rb

```diff
@@ -15,4 +20,12 @@
       expect(cart.reload.line_items).to be_empty
     end
   end
+
+  describe "#save_with_payment" do
+    it "creates an order with Stripe" do
+      expect do
+        order = Order.create(user: user, stripe_token: stripe_helper.generate_card_token)
+      end.to change{Order.count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.